mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/impl/HibernateOrmMassIndexingContext.java

```diff
@@ -173,10 +173,12 @@ public PojoMassIdentifierLoader createIdentifierLoader(PojoMassIndexingIdentifie
 					.collect( Collectors.toList() );
 
 			HibernateOrmQueryLoader<E, I> typeQueryLoader = createQueryLoader( typeContexts );
+			// NOTE: we are creating a regular and not a stateless session,
+			//   since a stateless session does not apply multitenant filter (for now)
```

Comment:
Maybe add a link to the Jira ticket in ORM?

Reply:
I've only opened a draft PR so far https://github.com/hibernate/hibernate-orm/pull/7675  🙈  let me create tickets ...